Resolves #1374 - Provide example of using context (#1411)
styleguidist/react-styleguidist: docs/Documenting.md

;<ModalExample />
````

> If component consumes React context, you need to have a context provider in the example or in a custom Wrapper. See [ThemeButton example](https://github.com/styleguidist/react-styleguidist/tree/master/examples/basic/src/components/ThemeButton).

Reviewer: ```suggestion
If a component consumes React Context, you need a context provider in the example or in a custom `Wrapper` component. See [ThemeButton example](https://github.com/styleguidist/react-styleguidist/tree/master/examples/sections/src/components/ThemeButton).
```
